x-pack/test/functional/apps/dataset_quality/dataset_quality_summary.ts

```diff
@@ -48,16 +48,14 @@ export default function ({ getService, getPageObjects }: DatasetQualityFtrProvid
   };
 
   describe('Dataset quality summary', () => {
-    before(async () => {
-      await synthtrace.index(getInitialTestLogs({ to, count: 4 }));
-      await PageObjects.datasetQuality.navigateTo();
-    });
-
     afterEach(async () => {
       await synthtrace.clean();
     });
 
     it('shows poor, degraded and good count as 0 and all dataset as healthy', async () => {
+      await synthtrace.index(getInitialTestLogs({ to, count: 4 }));
+      await PageObjects.datasetQuality.navigateTo();
+
```

Comment:
Those data were ingested for the first test in this suite only. Then, the `afterEach` hook will clean those data after the first test, and the second will ingest a different dataset. 
However, if we run in isolation the second test, it would still go through the `before` hook, which will make the test have an inconsistent data set for this test.
Ideally, tests like this should be written to run in isolation for faster debugging, this change only refactors this test suite but doesn't affect others.